frontend/javascripts/dashboard/publication_card.tsx

```diff
@@ -21,26 +26,50 @@ type ExtendedDatasetDetails = APIDatasetDetails & {
 };
 const thumbnailDimension = 500;
 const miniThumbnailDimension = 75;
-const blacklistedSegmentationNames = ["2012-09-28_ex145_07x2_ROI2017"];
```

Comment:
The problem was that this dataset only had an oversegmentation and was at the top of the list. Since the dataset is now much lower in the list and we have a version with applied mapping, this hotfix can go.